static/js/portico/signup.js

```diff
@@ -196,4 +196,40 @@ $(function () {
     $("body").on("click", "#choose_email .choose-email-box", function () {
         this.parentNode.submit();
     });
+
+    const offer_details_behavior = function () {
+        const selected_object = $("#organization_type_select")[0];
+        const selected_value = selected_object.options[selected_object.selectedIndex].value;
```

Comment:
Correct me if I'm wrong but doesn't `selected_object.value()` work?

Reply:
I guess you mean `selected_object.val()`. Yes, it will work. I was not much familiar with jquery at that time and I missed it after that I guess. I will change it.